test/nested-properties.test.js

```diff
@@ -0,0 +1,70 @@
+// Copyright IBM Corp. 2013,2016. All Rights Reserved.
+// Node module: loopback-datasource-juggler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// This test written in mocha+should.js
+'use strict';
+
+/* global getSchema:false, connectorCapabilities:false */
+var should = require('./init.js');
+
+var db, Model;
+
+describe('basic-querying', function() {
```

Comment:
Many of the test files in juggler are executed by our connectors too, in order to verify that CRUD operations work in those connectors as expected. I am not sure if it's worth creating a new file when you are adding just a single new test (I'd personally simply add the new test at the end of the existing `describe('find')` in [test/basic-querying.test.js](https://github.com/strongloop/loopback-datasource-juggler/blob/6bd9fca080d8050f9c89df4a62cdb4f57af99e45/test/basic-querying.test.js#L171-L918), possibly wrapped in a second-level `describe('nested properties')`).

If we decide to keep this new file, then:

 1. This `describe` block should be given a unique name, e.g. `query nested properties`.
 2. The new test file should be added to [test/common.batch.js](https://github.com/strongloop/loopback-datasource-juggler/blob/6bd9fca080d8050f9c89df4a62cdb4f57af99e45/test/common.batch.js) so that connectors pick it up.